packages/insomnia-app/app/ui/components/templating/tag-editor.js

```diff
@@ -252,6 +258,19 @@ class TagEditor extends React.PureComponent<Props, State> {
     this._update(this.state.tagDefinitions, tagDefinition, null, false);
   }
 
+  async _handleActionClick(action: NunjucksActionTag) {
+    const templateTags = await getTemplateTags();
+    const activeTemplateTag = templateTags.find(({ plugin, templateTag }) => {
+      return templateTag.name === this.state.activeTagData.name;
+    });
```

Comment:
Doesn't look like plugin is used any longer?

```suggestion
    const activeTemplateTag = templateTags.find(({ templateTag }) => {
      return templateTag.name === this.state.activeTagData.name;
    });
```